agent/src/index.ts

```diff
@@ -510,7 +510,10 @@ export async function createAgent(
 			...(getSecret(character, "COINBASE_API_KEY") &&
 			getSecret(character, "COINBASE_PRIVATE_KEY")
 				? [
-						coinbaseMassPaymentsPlugin,
+						...(getSecret(character, "COINBASE_MASS_PAYMENTS_ENABLED") ===
+						"true"
+							? [coinbaseMassPaymentsPlugin]
+							: []),
 						tradePlugin,
```

Comment:
We should add env for each of these and disable them all for prosper (that is trading)